dataland-frontend/tests/e2e/specs/framework-view-and-upload-journey/FrameworkSelection.ts

```diff
@@ -54,11 +40,9 @@ describe("The shared header of the framework pages should act as expected", { sc
         searchStringQueryParam: string
       ): void {
         cy.visit(`/companies?input=${searchStringQueryParam}&framework=${frameworkQueryParam}`);
```

Comment:
Why ist the old function interceptAndWaitForMetaDataRequest no longer necessary?

Reply:
It was only used to guarantee that the dropdown for framework selection is fully loaded. This is now solved differently:
The string 'Checking if' is searched on the page which is displayed if and only if the dropdown content is not loaded yet.

Reply:
Ok